knotx-core/knotx-knot-action/src/test/resources/fragment_form_self_out.txt

```diff
@@ -0,0 +1,8 @@
+    <form>
```

Comment:
I think that `ActionKnot` should not strip `<script>` tags. They are used later in `ViewKnot` to get `data-service` calls.